src/components/ProfilePage/AddressBox/AddressBox.tsx

```diff
@@ -51,17 +54,24 @@ export const AddressBox = ({ address, isEVM }: AddressBoxProps) => {
         <Image
           alt="Effigy Wallet Icon"
           src={
-            address && isEVM
-              ? `https://effigy.im/a/${address}.png`
-              : `https://effigy.im/a/${'jumper.eth'}.png`
+            imageLink
+              ? imageLink
+              : address && isEVM
+                ? `https://effigy.im/a/${address}.png`
+                : `https://effigy.im/a/${'jumper.eth'}.png`
           }
           width={128}
           height={128}
           style={{
+            backgroundColor: imageLink
+              ? theme.palette.mode === 'light'
+                ? '#F9F5FF'
+                : theme.palette.accent1Alt.main
+              : undefined,
```

Comment:
Maybe we could add some const and use it, to simplify the rough ternary operator:

`const defaultBackgroundColor = theme.palette.mode === 'light'? '#F9F5FF' : theme.palette.accent1Alt.main;`

Now: ->
`backgroundColor: imageLink ? defaultBackgroundColor : undefined;`